### ISSUE TITLE
npm install command?

### ISSUE THREAD

**Issue by mmendivilg** [marchah/node-mandrill#2]
I find your library very usefull, but I cannot find the npm install command.
Thank you

**Comment by marchah** [comment]
Not sure to understand what you mean?
How to add this library to your project?

Btw this lib is very outdated, i don't recommend using it

**Comment by mmendivilg** [comment]
Yes, how to add the library to the project, is there anything you would recommend for sending emails using mandrill?
thanks

**Comment by marchah** [comment]
you need to add the library as a dependency in your `packages.json` and do `npm|yarn install`

https://www.npmjs.com/package/@mailchimp/mailchimp_transactional looks to be the most recent lib but i'm still using https://www.npmjs.com/package/mandrill-api

**Comment by marchah** [comment]
actually looks like my library is using the latest version of https://www.npmjs.com/package/mandrill-api 😅
